frontend/package.json

```diff
@@ -80,5 +80,6 @@
     "tailwindcss": "^3.4.14",
     "typescript": "^5.5.4",
     "typescript-eslint": "8.12.2"
-  }
+  },
+  "packageManager": "pnpm@9.11.0+sha512.0a203ffaed5a3f63242cd064c8fb5892366c103e328079318f78062f24ea8c9d50bc6a47aa3567cabefd824d170e78fa2745ed1f16b132e16436146b7688f19b"
```

Comment:
Was this change intended?

Reply:
Nope, corepack randomly adds that and i forgot to remove it this time. I need to find a way to turn that off at some point.